### PR TITLE
Integrates bitstring status list

### PR DESCRIPTION
**Pull request by kezike**
This PR integrates the Bitstring Status List.

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
@kezike thanks! Reviewing now. (Can you fix the lint/indentation errors meanwhile?)

File: app/components/VerificationStatusCard/VerificationStatusCard.tsx [hunk 1]
```diff
@@ -80,7 +81,18 @@ export default function VerificationStatusCard({ credential, verifyPayload }: Ve
         <StatusItem
           positiveText="Has not been revoked by issuer"
           negativeText="Has been revoked by issuer"
-          verified={details[LogId.RevocationStatus]}
+          verified={
+            details[LogId.RevocationStatus] === undefined ||
+            details[LogId.RevocationStatus]
+          }
+        />
+        <StatusItem
```
Comment (dmitrizagidulin) [inline app/components/VerificationStatusCard/VerificationStatusCard.tsx:89]: Let's display this item conditionally -- only when there's a `credentialStatus` section. (Ideally, we'd only display it if the bitstring status credential had suspension type lists)

**Review by dmitrizagidulin — APPROVED** [review]
👍 looks good, pending:
* Fix linting errors on the Files Changed tab
* Display the 'suspended' line conditionally.

File: app/components/VerificationStatusCard/VerificationStatusCard.tsx (hunk 1 above)
Comment (kezike) [reply app/components/VerificationStatusCard/VerificationStatusCard.tsx:98]: @dmitrizagidulin I have changed this to display this section only if suspension is being tracked in the credential. For consistency, I am also only displaying revocation status if it is being tracked in the credential. Let me know what you think.

**Comment by kezike** [comment]
@dmitrizagidulin I have addressed all of your concerns and I also made additional modifications regarding validity period, so that the app can support both versions (`issuanceDate` vs. `validFrom` and `expirationDate` vs. `validUntil`).

File: app/components/VerificationStatusCard/VerificationStatusCard.tsx (hunk 1 above)
Comment (dmitrizagidulin) [reply app/components/VerificationStatusCard/VerificationStatusCard.tsx:98]: Thanks!

**Review by dmitrizagidulin — APPROVED** [review]

**Pull request merged**